Refs #32674 -- Noted that auto-created through table PKs cannot be au… (#14328)
django/django: docs/ref/settings.txt

Default primary key field type to use for models that don't have a field with
:attr:`primary_key=True <django.db.models.Field.primary_key>`.

> .. admonition:: Migrating auto-created through tables.

Reviewer: ```suggestion
.. admonition:: Migrating auto-created through tables
```

    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
    auto-created through tables for many-to-many relationships.

>     The primary key of auto-created through tables cannot currently be updated

Reviewer 2: I know it is implicit based on the previous paragraph, but it may be better to be very clear about this:
```suggestion
    The primary key of existing auto-created through tables cannot currently be updated
```

>     The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
>     auto-created through tables for many-to-many relationships.
> 
>     The primary key of auto-created through tables cannot currently be updated
>     by the migrations framework, however.

Reviewer: Maybe:
```suggestion
    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
    auto-created through tables for many-to-many relationships. However, the
    primary keys of existing auto-created through tables cannot currently be
    updated by the migrations framework
```
or even `Unfortunately` instead of `However` :thinking:

Author: Let's go with Unfortunately, but I'll push it as a new paragraph for visual emphasis. (We can collapse if you don't like it.)

    This means that if you switch the value of ``DEFAULT_AUTO_FIELD`` and then
    generate migrations, the primary keys of the related models will be

>     updated, as will the foreign keys from the through table, but the primary

Reviewer: I think we can chop `will` :think`
```suggestion
    updated, as the foreign keys from the through table, but the primary
```

Author: I think it is necessary. `as will` here ≈`same with` (but `same with` wouldn't work).

Author: I can think about a rephrase if you think it doesn't read.

If not, if likely my Britishness, and tendency to archaic forms, coming through — Happy to adjust!

Reviewer 2: > If not, it's likely my Britishness, and tendency to archaic forms, coming through — Happy to adjust!

It reads correctly to me as you've written it. I wouldn't have thought it archaic.

    key of the auto-created through table will not be migrated.

    In order to address this, you should add a
    :class:`~django.db.migrations.operations.RunSQL` operation to your
    migrations to perform the required ``ALTER TABLE`` step. You can check the
    existing table name through ``sqlmigrate``, ``dbshell``, or with the

>     field’s ``remote_field.through._meta.db_table`` property

Reviewer: ```suggestion
    field’s ``remote_field.through._meta.db_table`` property.
```

Author: Good spot. I'll pull in shortly. Thanks

    Note that explicitly defined through models are already handled by the

>     migrations system.

Reviewer: I would chop `Note that`.
